pkg/util/etcdutil/start_etcd.go

```diff
@@ -46,7 +46,7 @@ spec:
   containers:
   - command:
     - /bin/sh
-    - -c
+    - -ec
```

Comment:
Why do we need to wrap this in a shell here? I think we could execute this directly. FWIW when you use $(FOO) the kubelet will template in those values (they are not being used as environment variables - note the `"()"` vs `"{}"`